pkg/util/util.go

```diff
@@ -522,3 +524,18 @@ func IsGlobExpMatch(strToMatch string, globExps []string) (bool, error) {
 	}
 	return false, nil
 }
+
+// PrintMachineOutput prints json output
+func PrintMachineOutput(outputFlag string, resource interface{}) (string, error) {
+	var out []byte
+	var err error
+	switch outputFlag {
+	case "json":
+		out, err = json.Marshal(resource)
+	case "":
```

Comment:
do we need this case? I mean it's empty

Reply:
It would be a good idea to make this default maybe?
as in default: print this in not supported format